### PR TITLE
Sr/support page changes

### PR DESCRIPTION
**Pull request by SandraRodgers**

### CODE REVIEW

File: developers/src/pages/support/index.astro [hunk 1]
```diff
@@ -2,136 +2,186 @@
...
+			<div id="panels-section" class="col-span-2 order-first md:order-none">
+				<div class="panel">
+					<div class="heading relative">
+						<h3 class="title">Forums</h3>
+						<img class="h-[74px] absolute -top-4 right-4" src={alf} alt="DG Docs logo" />
+					</div>
+
+					<div class="text">
+						<div class="mb-4">
+							Our community forum is the official way to get help if you’re on a free account, and the wealth of information from Deepgram and our community make them a great
```
Comment (jpvajda) [inline developers/src/pages/support/index.astro:86]:
```suggestion
							Our community forum is the official way to get help if you’re on a free account and the wealth of information from Deepgram and our community make them a great
```

Comment (jpvajda) [reply developers/src/pages/support/index.astro:86]: I don't think we need a comma here.

File: developers/src/pages/support/index.astro [hunk 2]
```diff
@@ -2,136 +2,186 @@
...
+						<Link href="https://github.com/orgs/deepgram/discussions" icon="arrow-right" iconSuffix>View the community forum</Link>
+					</div>
+				</div>
+				<div class="panel">
+					<div class="heading relative">
+						<h3 class="title">Guides</h3>
+						<img class="h-[74px] absolute -top-4 right-4" src={codey} alt="DG Docs logo" />
+					</div>
+					<div class="text">
+						<div class="mb-4">Our guides walk you through step by step how the use most popular features of Deepgram.</div>
```
Comment (jpvajda) [inline developers/src/pages/support/index.astro:98]:
```suggestion
						<div class="mb-4">Our guides walk you through step by step how to use the most popular features of Deepgram.</div>
```

Comment (jpvajda) [reply developers/src/pages/support/index.astro:98]: fixing this sentence.

File: developers/src/pages/support/index.astro [hunk 3]
```diff
@@ -2,136 +2,186 @@
...
+					</div>
+				</div>
+				<div class="panel">
+					<div class="heading relative">
+						<h3 class="title">Docs</h3>
+						<img class="h-[74px] absolute -top-4 right-4" src={noodle} alt="DG Docs logo" />
+					</div>
+
+					<div class="text">
+						<div class="mb-4">Our extensive documentation show you how to use every part of Deepgram.</div>
```
Comment (jpvajda) [inline developers/src/pages/support/index.astro:109]:
```suggestion
						<div class="mb-4">Our extensive documentation shows you how to use every part of Deepgram.</div>
```

Comment (jpvajda) [reply developers/src/pages/support/index.astro:109]: fixing this sentence.

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: developers/src/pages/support/index.astro (hunk 1 above)
Comment (SandraRodgers) [reply developers/src/pages/support/index.astro:86]: Well it's two independent clauses, i.e. a compound sentence, so the former English teacher in me would disagree. But if you prefer the flow without the comma, I'll remove it!

**Review by jpvajda — APPROVED** [review]

**Pull request merged**